pkg/sink/sink.go

```diff
@@ -36,10 +42,15 @@ type Resource struct {
 	TriggersClient         triggersclientset.Interface
 	DiscoveryClient        discoveryclient.DiscoveryInterface
 	RESTClient             restclient.Interface
+	PipelineClient         pipelineclientset.Interface
 	EventListenerName      string
 	EventListenerNamespace string
 }
 
+const (
+	validateTaskWait = 20 * time.Second
```

Comment:
would something like `taskRunPollingInterval` be a more descriptive name for this value? 
(Also, this seems like a thing that we would want to make configurable in the future)

Reply:
Yes.